libs/shared/src/dto/topic/topic-subscriber.interface.ts

```diff
@@ -0,0 +1,15 @@
+import { EnvironmentId, ExternalSubscriberId, OrganizationId, SubscriberId, TopicId, TopicKey } from '../../types';
+
+export interface ITopicSubscriber {
```

Comment:
I couldn't guess what was the reason behind you creating this interface that is only used in the TopicSubscriberDTO. What it was? Do you plan to reuse it somewhere and we would need this interface for another use case?

Reply:
Please note that it is used in 
https://github.com/novuhq/novu/blob/517b4fa662a8b5a72d62a1518bff9d68c21f5909/packages/application-generic/src/usecases/get-topic-subscribers/get-topic-subscribers.use-case.ts#L48
as well.
Due to the reason, we do the get-topic-subscribers mapping in the application generic now, it does not make sense to use API TopicSubscriberDto
there.